launcher/src/test/scala/bloop/launcher/GlobalSettingsSpec.scala

```diff
@@ -0,0 +1,74 @@
+package bloop.launcher
+
+import java.nio.file.Files
+import scala.concurrent.Promise
+import bloop.bloopgun.util.Environment
+import bloop.internal.build.BuildInfo
+
+object GlobalSettingsSpec extends LauncherBaseSuite("1.3.2", BuildInfo.bspVersion, 9014) {
```

Comment:
Should we make this suite run and install the latest bloop version (`BuildInfo.version`) instead of `1.3.2`?
  
When you select the launcher with the latest version, it doesn't run by default unless you `install` it first in sbt. This step happens automatically on CI.
  
Testing the launcher works this way because we need to use the python script to install bloop and resolve its artifacts, and these resources are only generated during `install`.
  
We could probably update the `1.3.2` reference in `LauncherSpec` too. I imagine you copied this from there, we were testing the launcher spec for 1.3.2 because the launcher was released in 1.3.3 and I wanted to make sure that the launcher would work even if people used 1.3.2. We no longer need to guarantee that.

Reply:
I made the changes in `GlobalSettingsSpec` but left `LauncherSpec` unchanged since there is already a spec covering `BuildInfo.version`
```scala
object LatestStableLauncherSpec extends LauncherSpec("1.3.2")
object LatestMasterLauncherSpec extends LauncherSpec(BuildInfo.version)
```